libraries/redcore/api/soap/operation/operation.php

```diff
@@ -285,6 +285,7 @@ private function task($taskName, $data)
 
 		// We are setting the operation of the webservice to task
 		$this->webservice->options->set('task', $taskName);
+		JFactory::getApplication()->input->set('task', $taskName);
```

Comment:
I know this is already merged, but... wouldn't it have been better to get that task name manually from the webservice trying to access it?

Initially in that operation that Javi found the problem with, I started doing it while testing only via SOAP parameters, that's why it was expecting this `task` in the input, but when switching to JSON then the `task` is simply not provided, so I started fetching it in another way.

If you feel it's a good addition, it's ok, but in the end, it's not really necessary.